untrusted/urts/sys/Cargo.toml

```
@@ -14,6 +14,7 @@ mc-sgx-core-sys-types = { path = "../../../core/sys/types" }
 [build-dependencies]
 bindgen = "0.60.1"
 cargo-emit = "0.2.1"
+mc-sgx-core-build = { path = "../../../core/build" }
```

Nit: Add a newline to maintain separate sorted groups
```suggestion

mc-sgx-core-build = { path = "../../../core/build" }
```